src/Phansible/Resources/ansible/roles/rabbitmq/tasks/main.yml

```diff
@@ -0,0 +1,24 @@
+---
+- name: add rabbitmq official apt repository
+  apt_repository: repo='deb http://www.rabbitmq.com/debian/ testing main' state=present
+
+- name: add the verification key for the package
+  sudo: yes
+  shell: curl http://www.rabbitmq.com/rabbitmq-signing-key-public.asc | sudo apt-key add -
+
+- name: update repositories
+  apt: update_cache=yes
+
+- name: install rabbitmq
+  apt: pkg=rabbitmq-server state=installed force=yes
+
+- name: enable rabbitmq plugins
+  rabbitmq_plugin: names={{ item }} state=enabled
+  with_items: rabbitmq.plugins
+  when: rabbitmq.plugins is defined
+
+- name: add user
+  rabbitmq_user: user={{ rabbitmq.user }} password={{ rabbitmq.password }} node=rabbit@{{ vagrant_local.vm.hostname }} tags=administrator,{{ rabbitmq.user }} vhost=/ configure_priv=.* write_priv=.* read_priv=.* state=present
+
+- name: restart rabbitmq
+  service: name=rabbitmq-server state=reloaded
```

Comment:
This will restart rabbit every time even if changes are not performed.
Shouldn't be better to restart it only when required?